packages/sources/wrapped/src/endpoint/deposits.ts

```diff
@@ -0,0 +1,52 @@
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters, AxiosResponse } from '@chainlink/types'
+
+export const supportedEndpoints = ['deposits']
+
+export interface ResponseSchema {
+  [token: string]: Address[]
+}
+
+export type Address = {
+  address: string
+}
+
+const customError = (data: any) => {
+  return typeof data !== 'object'
+}
+
+export const inputParameters: InputParameters = {
+  symbol: false,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const symbol = validator.validated.data.symbol || 'ETH'
```

Comment:
Unless explicitly required otherwise, I'd like to require the symbol to be defined

```suggestion
  symbol: true,
}

export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
  const validator = new Validator(request, inputParameters)
  if (validator.error) throw validator.error

  const jobRunID = validator.validated.id
  const symbol = validator.validated.data.symbol
```